packages/notification-service/src/polling.ts

```diff
@@ -13,3 +14,14 @@ export const notificationPolling = AsyncPolling(async (end) => {
     end()
   }
 }, POLLING_INTERVAL)
+
+export const exchangePolling = AsyncPolling(async (end) => {
```

Comment:
I'm slightly concerned about the impact of having two things polling at the same time given that JS is single threaded, though admittedly this one doesn't run often. If you can confirm that it doesn't seem to interfere with the notification polling then I think we're good

Reply:
From running locally, this doesn't seem to interfere, but I'm going to test deploying this to integration to confirm